frontend/components/EditorPages/TimerSetter.tsx

```diff
@@ -0,0 +1,109 @@
+	const [isIncrementing, setIsIncrementing] = useState(false);
+	const [isDecrementing, setIsDecrementing] = useState(false);
+
+	// Store references to timeout and interval for holding -/+ buttons
+	let delayIntervalRef = useRef<NodeJS.Timeout>();
+	let intervalRef = useRef<NodeJS.Timer>();
+
+	// Incrementing/decrementing based on mouse down and mouse up events
+	useEffect(() => {
+		if (isIncrementing) {
+			// Click + button: increment by 1
+			setTimerValue((current) => current + 1);
+			// Hold + button: keep incrementing
+			delayIntervalRef.current = setTimeout(() => {
+				intervalRef.current = setInterval(() => {
+					setTimerValue((current) => current + 1);
+				}, 100);
+			}, 500);
+		} else if (isDecrementing) {
+			// Click - button: decrement by 1
+			setTimerValue((current) => current - 1);
+			// Hold - button: keep decrementing
+			delayIntervalRef.current = setTimeout(() => {
+				intervalRef.current = setInterval(() => {
+					setTimerValue((current) => current - 1);
+				}, 100);
+			}, 500);
+		} else {
+			// Clear timeout and interval if both isIncrementing and
+			// isDecrementing are set to false
+			clearTimeout(delayIntervalRef.current);
+			clearInterval(intervalRef.current);
+			delayIntervalRef.current = undefined;
+			intervalRef.current = undefined;
+		}
+	}, [isIncrementing, isDecrementing]);
+
+	// Stop if the minimum or maximum timer value is reached
+	useEffect(() => {
+		if (
+			(timerValue === MIN_TIMER_VALUE && isDecrementing) ||
+			(timerValue === MAX_TIMER_VALUE && isIncrementing)
+		) {
+			clearTimeout(delayIntervalRef.current);
+			clearInterval(intervalRef.current);
+			delayIntervalRef.current = undefined;
+			intervalRef.current = undefined;
+		}
+	}, [timerValue, isIncrementing, isDecrementing]);
+
+	function startIncrementing() {
+		if (timerValue < MAX_TIMER_VALUE) {
+			setIsIncrementing(true);
+		}
+	}
+
+	function stopIncrementing() {
+		setIsIncrementing(false);
+		onChange(timerValue);
+	}
+
+	function startDecrementing() {
+		if (timerValue > MIN_TIMER_VALUE) {
+			setIsDecrementing(true);
+		}
+	}
+
+	function stopDecrementing() {
+		setIsDecrementing(false);
+		onChange(timerValue);
+	}
+
+	return (
+		<div className="h-full bg-gray-100 flex flex-row items-center justify-between font-normal text-base rounded-lg lg:text-lg 2xl:text-xl">
+			<button
+				className="w-4 h-4 bg-white flex items-center justify-center text-red-200 rounded-md ml-1 my-1 hover:brightness-95 active:brightness-90 lg:w-5 lg:h-5 2xl:w-7 2xl:h-7"
+				onMouseDown={startDecrementing}
+				onMouseUp={stopDecrementing}
+			>
+				-
+			</button>
+
+			<div className="px-1">{timerValue}</div>
+
+			<button
+				className="w-4 h-4 bg-white flex items-center justify-center text-green-200 rounded-md mr-1 my-1 hover:brightness-95 active:brightness-90 lg:w-5 lg:h-5 2xl:w-7 2xl:h-7"
+				onMouseDown={startIncrementing}
+				onMouseUp={stopIncrementing}
+			>
+				+
+			</button>
+		</div>
+	);
+}
```

Comment:
Thanks for creating this, I had been thinking of a better way to allow bigger changes, pressing the button one at a time is definitely not ideal.
I do think there is some worth in allowing direct inputs on top of the buttons, with enforcement of the minimum and maximum of course.  Not necessary though.

Reply:
For the lack of a 404 page, I do think that we can add that, but it would be better to turn the upload page 404 into a component that we can use for pages like these, as that would work for any non-mobile pages. I think we could do that after merging, maybe.

Reply:
I agree with this, I think it would be easy to turn that 404 into a small separate component.
I could probably do that in the beginning of sprint 4 in a few minutes.